shortlinks/templates/search.html

```diff
@@ -0,0 +1,14 @@
+{{ template "z_header.html" .}}
+{{ template "form.html" .}}
```

Comment:
I don't really understand why you'd have the UI to create a new short link in the search results page?

Reply:
So the idea here is that you went to a page that doesn't exist that you thought existed, seems like a good time to add it since it was missing before.